policyengine_us/variables/gov/states/ne/tax/income/credits/ne_elderly_disabled_credit.py

```diff
@@ -0,0 +1,19 @@
+from policyengine_us.model_api import *
+
+
+class ne_elderly_disabled_credit(Variable):
+    value_type = float
+    entity = TaxUnit
+    label = "NE elderly/disabled tax credit"
+    unit = USD
+    definition_period = YEAR
+    reference = (
+        "https://revenue.nebraska.gov/files/doc/tax-forms/2021/f_1040n_booklet.pdf"
+        "https://revenue.nebraska.gov/files/doc/2022_Ne_Individual_Income_Tax_Booklet_8-307-2022_final_5.pdf"
+    )
+    defined_for = StateCode.NE
+
+    def formula(tax_unit, period, parameters):
+        credit = tax_unit("elderly_disabled_credit", period)
```

Comment:
(This initially confused me since I misread as state tax capping, in which case it would be capped at the stage of summing all non-refundable credits)
```suggestion
        # "If the federal credit has been limited by your federal tax liability, use the lesser amount."
        credit = tax_unit("elderly_disabled_credit", period)
```

Reply:
Clarified with a local variable name change.